test/integration/webhook/core/admissioncheck_test.go

```diff
@@ -45,20 +45,14 @@ var _ = ginkgo.Describe("AdmissionCheck Webhook", func() {
 			),
 		)
 
-		ginkgo.DescribeTable("Validate AdmissionCheck on creation", func(ac kueue.AdmissionCheck, errorType int) {
+		ginkgo.DescribeTable("Validate AdmissionCheck on creation", func(ac kueue.AdmissionCheck, matcher gomega.OmegaMatcher) {
```

Comment:
```suggestion
		ginkgo.DescribeTable("Validate AdmissionCheck on creation", func(ac kueue.AdmissionCheck, matcher gomega.GomegaMatcher) {
```
Looks like OmegaMatcher is deprecated.

Reply:
Oh, I see. Good catch.

Reply:
Done.